easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb

```diff
@@ -10,15 +10,16 @@ description = """The JasPer Project is an open-source initiative to provide a fr
 toolchain = {'name': 'intel', 'version': '2016b'}
 toolchainopts = {'pic': True}
 
-sources = [SOURCELOWER_TAR_GZ]
 source_urls = ['http://www.ece.uvic.ca/~frodo/jasper/software/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['54d85428e35263642358a11c312d61cbc054170546fae780e11271df5d1502e8']
 
-builddependencies = [('CMake', '3.6.1')]
+builddependencies = [('CMake', '3.7.1')]
```

Comment:
@vanzod Why change the CMake here?

Reply:
Just because I cannot build and generate a report with `CMake/3.6.1` on Debian 9.
I can revert it back now that a report is there.

Reply:
Why? CMake 3.6.1 doesn't build on Debian 9?

I prefer stick to our policy of not bumping dependency versions, although here it's not a huge issue since it's only a build dep...

Reply:
Actually, nevermind, if this helps on Debian 9, it's fine to bump the version in this particular case since it can't introduce any conflicts.